src/AddImageChecks.cpp

```diff
@@ -650,6 +658,11 @@ Stmt add_image_checks(Stmt s,
         s = LetStmt::make(lets_required[i - 1].first, lets_required[i - 1].second, s);
     }
 
+    // Inject the code that checks that does msan checks. (Note that this ignores no_asserts.)
```

Comment:
There are people running with msan-no_asserts ?!

Reply:
I haven't seen that yet, but I will bet you $1 someone does at some point (even inadvertently)...